pkg/autoscaler/config_test.go

```diff
@@ -115,6 +117,8 @@ func TestNewConfig(t *testing.T) {
 			c.MaxScaleDownRate = 3
 			c.MaxScaleUpRate = 1.01
 			c.StableWindow = 5 * time.Minute
+			c.EnableScaleToZero = true
```

Comment:
do we need this change?

Reply:
actually not needed because it's included in the default config